lib/cqm/converter/bonnie_measure.rb

```diff
@@ -155,6 +154,19 @@ def shallow_copy(bonnie_measure)
   cqm_measure
 end
 
+def convert_source_data_criteria(bonnie_measure)
+  @map_definition_and_status_to_model ||= JSON.parse(File.read(File.join(File.dirname(__FILE__), 'map_definition_and_status_to_model.json')))
+  converted_source_data_criteria = bonnie_measure.source_data_criteria.map do |_, sdc|
+    key = "#{sdc['definition']}::#{sdc['status']}"
+    model_name = @map_definition_and_status_to_model[key]['model_name']
```

Comment:
Suggest checking if something at `key` exists before continuing. If it doesn't exist then raise an error detailing the type that was not found. This would allow our batch script to log failed conversions.